examples/kitchen-sink.yaml

```diff
@@ -27,16 +27,16 @@ logger_provider:
   processors:
     - # Configure a batch log record processor.
       batch:
-        # Configure delay interval (in milliseconds) between two consecutive exports.
+        # Configure delay interval (in milliseconds) between two consecutive exports. Value must be > 0.
         # If omitted or null, 1000 is used.
         schedule_delay: 5000
```

Comment:
Arguably a schedule delay of 0 is valid. Made this change anyway to prompt debate.

Reply:
I agree with your comment about 0 being valid https://github.com/open-telemetry/opentelemetry-specification/pull/4331#discussion_r1878484937